datasets/flores-200/metadata.json

```diff
@@ -215,6 +231,152 @@
           }
         }
       ]
+    },
+    {
+      "@type": "ml:RecordSet",
```

Comment:
Note: An alternate approach would have been to specify `"split": "dev" | "devtest"` as a parameter of all ml:RecordSets in order not to repeat them.

But because filtering is not implemented yet in mlcroissant, your current approach is probably the best.